pyproject.toml

```
@@ -1,15 +1,96 @@
 [build-system]
 requires = [
-    "setuptools>=42",
+    "scikit-build-core",
+    "swig",
     "oldest-supported-numpy; python_version < '3.9'",
     "numpy>=2.0.0; python_version >= '3.9'",
 ]
-build-backend = "setuptools.build_meta"
+build-backend = "scikit_build_core.build"
 
+[project]
+name = "wasserstein"
+version = "1.1.0"
+description = "Python package wrapping C++ code for computing Wasserstein distances"
+readme = "README.md"
+license.file = "LICENSE"
+requires-python = ">= 3.7"
+authors = [
+    { name = "Patrick T. Komiske III", email = "pkomiske@mit.edu" },
+]
```

```suggestion
]
maintainers = [ {name = "Jordan Ashley", email = "jordan.ashley@cern.ch"} ]
```

For sure have earned that.

---

That's awesome. 😄